src/components/appComponents/booksComponents/AddBooForm.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function AddBookForm() {
+  return (
+    <div>
+      <h2>Add  NEW BOOK</h2>
+      <form>
+        <input type="text" name="title" placeholder="Book title" />
+        <button type="button" className="add-btn btn-primary">ADD BOOK</button>
```

Comment:
- [ ] Please, include an input for Author too (and include all the structure to redux for this too). This will help to add more information to every book. 🐶

Reply:
Thank you for review.

Reply:
An input for Author was added.